alpaca_trade_api/rest_async.py

```diff
@@ -0,0 +1,183 @@
+import os
+import aiohttp
```

Comment:
aiohttp is missing from the requirements.txt!

Reply:
added